sklearn/linear_model/_base.py

```diff
@@ -501,6 +505,26 @@ def fit(self, X, y, sample_weight=None):
         self : returns an instance of self.
         """
 
+        if self.normalize != "deprecate":
+            if not self.normalize:
+                warnings.warn(
+                    "'normalize' was deprecated in version 0.24 and will be"
+                    " removed in 0.26.", FutureWarning
+                )
+            else:
+                warnings.warn(
+                    "'normalize' was deprecated in version 0.24 and will be"
+                    " removed in 0.26. If you wish to keep an equivalent"
+                    " behaviour, use  Pipeline with a StandardScaler in a"
+                    " preprocessing stage:"
+                    "  model = make_pipeline( \n"
+                    "    StandardScaler(with_mean=False), \n"
```

Comment:
```suggestion
                    "    StandardScaler(), \n"
```
the with_mean=False  is not necessary. As your test shows it's equivalent when it comes to prediction which is what matters.